### ISSUE TITLE
Move "build in ACR" command to ACA extension

### ISSUE THREAD

**Issue by bwateratmsft** [microsoft/vscode-docker#3873]
The Azure Container Apps extension is soon adding a command to build images in ACR. Using this would allow us to remove several Azure packages as dependencies, improving load time and reducing bundle size.

TODO: do we want to remove the command entirely, or leave in a soft entry point like we currently have with "Deploy to ACA"?

**Comment by dbreshears** [comment]
Will this require having an ACA dependency to simply deploy to ACR?

**Comment by bwateratmsft** [comment]
> Will this require having an ACA dependency to simply deploy to ACR?

No, the only thing I'd like to move is the "Run as task in Azure" / "Build image in Azure" feature. Local build and push, along with the other ACR features (repo browser essentially) should definitely stay in the Docker extension.

**Comment by dbreshears** [comment]
Let's seperate this into a seperate NPM package that ACA and other extensions can leverage

**Comment by dbreshears** [comment]
Moving to future as ACA still requires the Docker extension for authenticating and getting subscription/registry information.
